src/components/shared/navbar/NavBarUserActions.tsx

```diff
@@ -0,0 +1,45 @@
+import { IconButton, Stack, Tooltip } from '@mui/material'
+import React from 'react'
+import { Link } from 'react-router-dom'
+import useAuth from '../../../hooks/use-auth'
+import { useNavigate } from 'react-router'
+import ThemeSwitcher from './ThemeSwitcher'
+import LogoutIcon from '@mui/icons-material/Logout'
+import MeetingRoomIcon from '@mui/icons-material/MeetingRoom'
+import LoginIcon from '@mui/icons-material/Login'
+
+const NavBarUserActions = () => {
+  const { email, signOut } = useAuth()
+  const navigate = useNavigate()
+
+  if (!email) {
+    return (
+      <Stack spacing={2} direction={'row'}>
+        <Tooltip title={'Create a free account'}>
```

Comment:
Not really a fan of icons only stuff as icons need to be super specific and universal so the user intuitively understands what the icon does. I didn't get the MeetingRoomIcon though XD I think we should revisit this one. ;)